frontend/lib/components/utils/createTask/then_that_button.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:frontend/components/utils/createTask/add_button.dart';
+
+class ThenThatButton extends StatelessWidget {
+  const ThenThatButton({
+    this.enabled = true,
+    Key? key,
+  }) : super(key: key);
+
+  final bool enabled;
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      color: Colors.black.withOpacity(enabled ? 0.9 : 0.4),
+      borderRadius: const BorderRadius.all(Radius.circular(7)),
+      child: InkWell(
+        splashColor: Colors.white.withOpacity(0.2),
+        highlightColor: Colors.transparent,
+        onTap: enabled
+            ? () {
+                // Get.to(
+                //   TaskCardAbout(task: task),
+                //   transition: kIsWeb
+                //       ? Transition.noTransition
+                //       : Transition.rightToLeft,
+                // );
```

Comment:
Same thing here, remove this unused comment.

Reply:
fixed in [6c55f91](https://github.com/Cleopha/ifttt-like/pull/66/commits/6c55f914bbd793c6159f804eafd9b291aaed715b)